### PR TITLE
#VP-1417 Upgrade to .net core 3.1

### PR DESCRIPTION
**Pull request by kostyrin**

### CODE REVIEW

File: VirtoCommerce.Storefront.Model/VirtoCommerce.Storefront.Model.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (kostyrin) [inline VirtoCommerce.Storefront.Model/VirtoCommerce.Storefront.Model.csproj]: not sure that right use .Web, it is a library.

File: VirtoCommerce.LiquidThemeEngine/VirtoCommerce.LiquidThemeEngine.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (kostyrin) [inline VirtoCommerce.LiquidThemeEngine/VirtoCommerce.LiquidThemeEngine.csproj]: not sure that right use .Web, it is a library.

File: VirtoCommerce.Storefront/VirtoCommerce.Storefront.csproj [hunk 3]
```diff
@@ -64,12 +64,12 @@
...
     <PackageReference Include="PagedList.Core" Version="1.17.4" />
     <PackageReference Include="Polly" Version="7.2.0" />
-    <PackageReference Include="Scrutor" Version="3.1.0" />
+    <PackageReference Include="Scrutor" Version="3.2.0" />
     <PackageReference Include="StackExchange.Redis" Version="2.0.601" />
     <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="5.0.0" />
     <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="5.0.0" />
 	<PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="5.0.0" />
-    <PackageReference Include="VirtoCommerce.Tools" Version="1.2.0" />
+    <PackageReference Include="VirtoCommerce.Tools" Version="3.0.0-rc.3" />
```
Comment (kostyrin) [inline VirtoCommerce.Storefront/VirtoCommerce.Storefront.csproj]: are you testing after the changing?

File: VirtoCommerce.Storefront.Model/VirtoCommerce.Storefront.Model.csproj (hunk 1 above)
Comment (akak1977) [reply VirtoCommerce.Storefront.Model/VirtoCommerce.Storefront.Model.csproj]: You're right. I made changes.

File: VirtoCommerce.LiquidThemeEngine/VirtoCommerce.LiquidThemeEngine.csproj (hunk 2 above)
Comment (akak1977) [reply VirtoCommerce.LiquidThemeEngine/VirtoCommerce.LiquidThemeEngine.csproj]: You're right. I made changes.

File: VirtoCommerce.Storefront/VirtoCommerce.Storefront.csproj (hunk 3 above)
Comment (akak1977) [reply VirtoCommerce.Storefront/VirtoCommerce.Storefront.csproj]: Yes. It works well.

File: VirtoCommerce.LiquidThemeEngine/VirtoCommerce.LiquidThemeEngine.csproj [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <Version>4.1.7</Version>
     <Description>The storefront implementation of the Virto Commerce platform.</Description>
     <Copyright>Copyright © 2011-2019 Virto Commerce - an enterprise Microsoft ecommerce platform ©. All rights reserved</Copyright>
```
Comment (OlegoO) [inline VirtoCommerce.LiquidThemeEngine/VirtoCommerce.LiquidThemeEngine.csproj]: 2011-2020

**Comment by akak1977** [comment]
Technically closed, because equal PR was merged to branch dev-3.0.0

**Pull request closed without merging**